extensions/npm/main.ts

```diff
@@ -176,7 +176,13 @@ async function checkDryRun(subcommand: string, args: string[]) {
     await abort(status.code);
   }
 
-  const lockfile = await PhylumApi.parseLockfile("./package-lock.json", "npm");
+  let lockfile;
+  try {
+    lockfile = await PhylumApi.parseLockfile("./package-lock.json", "npm");
+  } catch (_e) {
+    console.warn(`[${yellow("phylum")}] No lockfile created.\n`);
+    return;
```

Comment:
You can probably exit here instead of return.

```suggestion
    await abort(0);
```

Reply:
I don't think we should, since we did not run the proper command yet at this point.